docs/.vuepress/theme/components/Frameworks.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <nav class="nav-versions nav-links" v-if="isFrameworked">
+    <img :src="imageUrl"/>
+    <!-- user links -->
+    <nav class="nav-item" >
+      <DropdownLink :item="item"></DropdownLink>
+    </nav>
+  </nav>
+</template>
+
+<script>
+import DropdownLink from '@theme/components/DropdownLink.vue';
+
+const frameworkIdToFullName = new Map([
+  ['javascript', 'JavaScript'],
+  ['react', 'React'],
+  ['angular', 'Angular'],
+  ['vue2', 'Vue 2'],
+  ['vue3', 'Vue 3'],
+]);
+
+export default {
+  name: 'Frameworks',
```

Comment:
The `Frameworks` name for a component looks too generic to me. I know it's similar to the other dropdown, `Versions`, but I'm not a huge fan of that one either. Something like `FrameworkDropdown` or something along these lines would be more fitting, what do you think?